### PR TITLE
Omar/swift5

### PR DESCRIPTION
**Pull request by Omarrasheed**
#215 
- Migrate to Swift 5
- Remove unnecessary pods
- Run Swiftlint Autocorrect, add it to build script, and fix some of the easier warnings
- Will tackle the rest of the warnings in another issue next semester

### CODE REVIEW

**Review by ywsang — APPROVED** [review]

File: .swiftlint.yml [hunk 1]
```diff
@@ -18,16 +18,17 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
 force_cast: warning # implicitly. Give warning only for force casting
 
 force_try:
-severity: warning # explicitly. Give warning only for force try
+  severity: warning # explicitly. Give warning only for force try
```
Comment (mattbarker016) [inline .swiftlint.yml:21]: Should this be on the same line, or is it like an if block?

**Comment by mattbarker016** [comment]
Were there any big changes other than spacing stuff? A bit hard to parse 😅

**Comment by Omarrasheed** [comment]
Not rly lmao most of it was swiftlint autocorrect. The main things r that I changed the proj to be on swift5 now. Migration was smooth and no errors. There were like a depracated function in extensions for nsrange I think but that was about the extent of those fixes. I also took out fuzzy_wuzzy swift

File: .swiftlint.yml (hunk 1 above)
Comment (Omarrasheed) [reply .swiftlint.yml:21]: I think this can be on the same line. I think this was taken from their demo code and they were showing the different ways the settings can be changed. I'll put it on the same line

Comment (mattbarker016) [reply .swiftlint.yml:21]: Was just curious!

**Comment by mattbarker016** [comment]
Did you make these changes yourself, or it did them all?

```
    func nsRange(from range: Range<String.Index>) -> NSRange {
	        let from = range.lowerBound.encodedOffset
	        let to = range.upperBound.encodedOffset
	        return NSRange(location: from - startIndex.encodedOffset, length: to - from)
	        let from = range.upperBound.utf16Offset(in: self)
	        let to = range.upperBound.utf16Offset(in: self)
	        return NSRange(location: from - startIndex.utf16Offset(in: self), length: to - from)
	    }
```

**Comment by Omarrasheed** [comment]
I had to change them manually

**Comment by mattbarker016** [comment]
One should be lowerBound and one should be upperBound. Look at the original code.

**Comment by Omarrasheed** [comment]
Good catch thank you!

**Comment by mattbarker016** [comment]
If you think there could be any other sneaky little changes like that, give it one more once over. Otherwise, looks good to me.
